MicroBenchmarks/LoopVectorization/CMakeLists.txt

```diff
@@ -14,6 +14,7 @@ llvm_test_executable(LoopVectorizationBenchmarks
   MathFunctions.cpp
   RuntimeChecks.cpp
   VectorOperations.cpp
+  LoopInterleaving.cpp
```

Comment:
It would probably be better to have this as a separate benchmark binary, as here are a lot of benchmarks added in this file and running it takes a while